### PR TITLE
chore(design-tokens): DSYS-819 | update typography tokens

### PR DESCRIPTION
**Pull request by sirineJ**
Addresses [DSYS-819](https://sumupteam.atlassian.net/browse/DSYS-819)

## Purpose

When implementing [DSYS-711](https://sumupteam.atlassian.net/browse/DSYS-711), the design token values weren’t finalized yet, so I used preliminary values. 

Once the values have been finalized in [Mapping Typography](https://docs.google.com/spreadsheets/d/1LGH5LjRdsC_hl56YPxKa5Tp-PQsWuJ9x9uMfuphtzVI/edit?gid=1990681080#gid=1990681080), we need to update them in the web implementation.

## Approach and changes

- Check if the token structure can be simplified: for example, do we need component and size-specific tokens for letter-spacings?

- Update the token values in [circuit-ui/packages/design-tokens/themes/shared.ts at next · sumup-oss/circuit-ui](https://github.com/sumup-oss/circuit-ui/blob/next/packages/design-tokens/themes/shared.ts#L162-L508) 
 
- Remove the `-typography` infix from typography tokens'names

## Definition of done

* [ ] Development completed
* [ ] Reviewers assigned
* [ ] Unit and integration tests
* [ ] Meets minimum browser support
* [ ] Meets accessibility requirements

### CODE REVIEW

File: packages/circuit-ui/components/SubHeadline/SubHeadline.mdx [hunk 1]
```diff
@@ -18,4 +18,4 @@ The SubHeadline component helps break up larger related chunks of content in the
 
 ## Accessibility
 
-All accessibility guidelines for the [Headline component](Typography/Headline) also apply to the SubHeadline component.
+All accessibility guidelines for the [Headline component](Typography/Headline) also apply to the [Display component](Typography/Display) component.
```
[the remark this answers is not in the document]
Comment (sirineJ) [reply packages/circuit-ui/components/SubHeadline/SubHeadline.mdx:21]: oups

File: docs/features/1-theme.mdx [hunk 2]
```diff
@@ -61,9 +60,16 @@ Use spacings for gutters, margins, and paddings. Don't use it for border width,
...
 
-<CustomPropertiesTable namespace="typography" preview={Typography} />
+<CustomPropertiesTable namespace="[
+    'body',
+    'display',
+    'headline',
+    'compact',
+    'numeral',
+    'typography',
+  ]" preview={Typography} />
```
Comment (connor-baer) [inline docs/features/1-theme.mdx:72]:
```suggestion
<CustomPropertiesTable 
  namespace={[
    'body',
    'display',
    'headline',
    'compact',
    'numeral',
    'typography',
  ]}
  preview={Typography} 
/>
```

File: packages/circuit-ui/components/Body/Body.module.css [hunk 3]
```diff
@@ -1,21 +1,22 @@
+.base {
```
Comment (connor-baer) [inline packages/circuit-ui/components/Body/Body.module.css:1]: The `.base` class name needs to be applied to the element in `Body.tsx`.

File: packages/circuit-ui/components/Compact/Compact.module.css [hunk 4]
```diff
@@ -1,21 +1,22 @@
+.base {
```
Comment (connor-baer) [inline packages/circuit-ui/components/Compact/Compact.module.css:1]: Same here – you need to apply the new class name.

File: docs/features/1-theme.mdx
Comment (connor-baer) [inline docs/features/1-theme.mdx:1]: It looks like this file was formatted with different settings from [Biome's formatting defaults](https://biomejs.dev/formatter/#options) 🤔  

Tab-width should be 2 spaces and there should be a space between the props and JSX closing tag.

**Review by connor-baer — APPROVED** [review]

**Pull request merged**